adapters/tappx/tappx.go

```diff
@@ -39,6 +39,7 @@ type tappxParams struct {
 	Host     string `json:"host"`
 	TappxKey string `json:"tappxkey"`
 	Endpoint string `json:"endpoint"`
+	BidFloor string `json:"bidfloor"`
 }
```

Comment:
Unless I'm wrong, the `tappxParams struct` doesn't seem to be used anywhere in your code. Are we sure we need it at all?